yapapi/storage/gftp.py

```diff
@@ -280,12 +308,8 @@ def __init__(self, *, tmpdir: Optional[str] = None):
         self._lock: asyncio.Lock = asyncio.Lock()
 
         # Flag indicating if this `GftpProvider` will close unpublished URLs.
-        # If set to `True` then the provider will call `gftp close <URL>` for an URL
-        # that has no longer any published files. This should be the default behavior,
-        # but it may cause errors, due to a bug in `gftp` prior to version `0.7.3`
-        # (see https://github.com/golemfactory/yagna/pull/1501), and is therefore turned
-        # on only if `read_use_gftp_close_env_var()` returns `True`.
-        self._close_urls: bool = read_use_gftp_close_env_var() or False
+        # See this class' docstring for more details.
+        self._close_urls: Optional[bool] = read_use_gftp_close_env_var()
```

Comment:
Late to the party here! How about renaming this to `_read_gftp_close_env_var`? I know currently it includes more of the env var's name, but the double verb is somewhat confusing.